### DISCUSSION TITLE
Welcome to SixtyFPS UI Discussions!

### DISCUSSION THREAD

**Discussion by tronical** [slint-ui/slint#80] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by Taha-Firoz** [comment]
Whats the exact benefit of using sixty fps over more mature technologies like QT's QML, I do enjoy the syntax of sixty fps however libraries like LVGL have shown to be able to provide similar compile targets and are more mature. This isn't any way to put down all the effort the SixtyFPS team has put in but it's just some information that's missing from the overall projects documentation.

**Comment by hunger** [comment]
@Taha-Firoz: Qt QML and LVGL are two very different beasts:-)

I can not provide an "official" answer, so I can only list some of the things I personally find interesting about sixtyfps:

* The type-safe language that gets compiled down to native code
* The limits build into the language: It can not do everything and the kitchen sink -- which should enable great tooling in the mid-term.
* The safety sixtyfps gets due to being written in rust. That should e.g. eliminate most memory issues.
* Focused on UI development only
* Can be used with multiple languages: C++, rust and node JS right now. It should be easy to adapt for even more languages in the future.
* Finally Olivier and Simon both have a lot of experience designing UI libraries. I trust them to do the right thing.

But yes, you will find more features and more (API/ABI) stability elsewhere at this time.
